modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java

```diff
@@ -300,19 +293,15 @@ private void displayInternalServerError() {
                                             null);
 		alertDialog.show();
 	}
-	
+
 	/**
 	 * Clears application data and displays unregister success message.
 	 */
 	private void clearAppData() {
 		CommonUtils.clearAppData(context);
-
-		alertDialog = CommonDialogUtils.getAlertDialogWithOneButtonAndTitle(context,
-                                            getResources().getString(R.string.title_head_registration_error),
-                                            getResources().getString(R.string.error_for_all_unknown_registration_failures),
-                                            getResources().getString(R.string.button_ok),
-                                            isRegisteredFailedOKBtnClickListerner);
-		alertDialog.show();
+        	if (Constants.DEBUG_MODE_ENABLED) {
+            		Log.d(TAG, "App data cleared");
```

Comment:
Wrong grammar. Should ideally be something like "Application data is/has been cleared"